Client/config/settings.py

```
@@ -0,0 +1,139 @@
+                # Assuming model_dir is relative to project root (parent of Client/)
+                model_dir = Path(__file__).parent.parent.parent / model_dir
+
+            alt_path = model_dir / model_path
+            if alt_path.exists():
+                self.settings["model_path"] = str(alt_path.absolute())
+            elif model_path.exists():
+                self.settings["model_path"] = str(model_path.absolute())
+            else:
+                raise ValueError(f"Configuration Error: model_path '{model_path}' does not exist.")
+        elif not model_path.exists():
+             raise ValueError(f"Configuration Error: model_path '{model_path}' does not exist.")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)